Library/Formula/pxz.rb

```diff
@@ -34,3 +34,16 @@ def install
  		if (m == MAP_FAILED) {
  			perror("mmap failed");
  			exit(EXIT_FAILURE);
+diff --git a/Makefile.orig b/Makefile
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!